packages/@ourworldindata/grapher/src/chart/ChartManager.ts

```diff
@@ -24,6 +25,7 @@ import { ColorScale } from "../color/ColorScale"
 // The possible options common across our chart types. Not all of these apply to every chart type, so there is room to create a better type hierarchy.
 
 export interface ChartManager {
+    base?: React.RefObject<SVGGElement | HTMLDivElement>
```

Comment:
This seems to be consistently present across chart types—is it actually the correct way to get the parent ref?